packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -12,6 +12,13 @@ import { nls } from '../messages';
 import { telemetryService } from '../telemetry';
 import { getRootWorkspacePath, hasRootWorkspace, OrgAuthInfo } from '../util';
 
+const validManageableStates = new Set([
+  'unmanaged',
+  'installedEditable',
+  'deprecatedEditable',
+  undefined // not part of a package
+]);
```

Comment:
are these the same ones listed under `manageableStates` on this page? https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_retrieveresult.htm 
would we need to support any of the other ones listed there?